### PR TITLE
chore: add extendedDebugInfo tip to debug tool docs

### PR DESCRIPTION
**Pull request by mcollovati**

### CODE REVIEW

**Review by MatthewVaadin — CHANGES_REQUESTED** [review]

File: articles/tools/azure/session-replication-debug-tool.adoc [hunk 1]
```diff
@@ -8,6 +8,13 @@ order: 110
 
 To help making HTTP session fully serializable and deserializable, Kubernetes Kit offers a tool whose aim is to discover main issues during development.
 
+.Enable serialization extended debug information
+[TIP]
+When facing exceptions on serialization phase, it may be useful set the `sun.io.serialization.extendedDebugInfo` system property to make Java serialization verbose.
+However, be aware that to represent the objects beans serialized, the `toString()` method is used, and this in rare cases may cause issues not related to serialization.
```
[the remark this answers is not in the document]
Comment (mcollovati) [reply articles/tools/azure/session-replication-debug-tool.adoc:14]: Thanks, I'll apply the same also on the session replication page

[the remark this answers is not in the document]
Comment (mcollovati) [reply articles/tools/azure/session-replication-debug-tool.adoc]: done

**Comment by heruan** [comment]
Should we use AsciiDoc's [`include`](https://docs.asciidoctor.org/asciidoc/latest/directives/include/) directive to have the admonition text in a single external file and use it in both pages?

**Comment by mcollovati** [comment]
> Should we use AsciiDoc's [`include`](https://docs.asciidoctor.org/asciidoc/latest/directives/include/) directive to have the admonition text in a single external file and use it in both pages?

Good point. I'll give it a try

File: articles/tools/azure/session-replication-debug-tool.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (jouni) [reply articles/tools/azure/session-replication-debug-tool.adoc]: Seems like a bit too complex sentence structure to me ("however, be aware that, in order to"), but let’s do the language clean up in the main branch.

**Review by MatthewVaadin — APPROVED** [review]

**Review by heruan — APPROVED** [review]

**Pull request merged**